### ISSUE TITLE
Ticketstatus stays on "new" even after replying (Docker 10_1 to 11_0 Upgrade)

### ISSUE THREAD

**Issue by shad00m** [RotherOSS/otobo#3696]
We recently upgraded our Docker Instance of OTOBO 10_1 to 11_0. (11.0.5)
After replying to a ticket with the status "new" it used to change to "open" this does not happen anymore.
This causes the ticket to stay inside the "New Tickets" widget, even after an Agent has replied to it already.
I have tried to enable the Ticket::EventModulePost###2800-ForceStateChangeOnLock to force set it to "open" but nothing changes. It always stays on new.
![image](https://github.com/user-attachments/assets/6491dfb2-7e10-4ce7-b4d1-c70d56cf1e52)

There are no errors or anything in the docker logs when replying to a ticket. I apologize for creating an issue ticket without logs, but there's nothing i could find.

**Comment by wornet-aer** [comment]
Is the full docker-compose stack running? Especially the daemon could be on-duty for changing this, but I'm not quite sure about that. We moved to 11.x yesterday and didn't have any issues with stuck ticket-states yet.

The docker logs are not that verbose from my experience. Have you checked the logs section in the OTOBO Admin panel?

**Comment by shad00m** [comment]
The full docker compose stack was up and marked as healthy.
![image](https://github.com/user-attachments/assets/1c46fcf6-0364-4682-b020-9245201bc55f)

I did a quick `docker compose down && docker compose up -d` even though I have already restarted multiple times after the upgrade... now it changes the ticketstatus properly, I'll close this ticket and observe the behaviour for a while. I'll reply here if it stops changing the status randomly again.

Thank you so much for the hint.
